### PR TITLE
testing/shairport-sync: upgrade to 3.0.2 / Add init script

### PR DESCRIPTION
**Pull request by henriknil**
testing/shairport-sync: upgrade to 3.0.2 / Add init script

### CODE REVIEW

**Comment by kaniini** [comment]
first patch has an incorrect author line, please fix it

**Comment by henriknil** [comment]
Updated with correct author line and removed a commented line in the open-rc script

File: testing/shairport-sync/shairport-sync.initd [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/sbin/openrc-run
+
+description="Implements a synchronous (multi-room-capable) AirPlay receiver"
+name="shairport-sync $SVCNAME"
```
Comment (jirutka) [inline testing/shairport-sync/shairport-sync.initd]: `$SVCNAME` is deprecated, it should be named `$RC_SVCNAME`. Please read [openrc-run(8)](http://manpages.org/openrc-run/8).

File: testing/shairport-sync/shairport-sync.initd [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+command="/usr/bin/shairport-sync"
+command_args="-d"
+pidfile="/var/run/$SVCNAME.pid"
```
Comment (jirutka) [inline testing/shairport-sync/shairport-sync.initd]: It should be `/run/$RC_SVCNAME.pid`.

File: testing/shairport-sync/shairport-sync.initd [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#!/sbin/openrc-run
+
+description="Implements a synchronous (multi-room-capable) AirPlay receiver"
+name="shairport-sync $SVCNAME"
+command="/usr/bin/shairport-sync"
+command_args="-d"
```
Comment (jirutka) [inline testing/shairport-sync/shairport-sync.initd]: It’s OpenRC’s job to daemonize started process, letting the process daemonize itself is a bad practice. Please remove this option and declare `command_background="yes"`.

File: testing/shairport-sync/shairport-sync.initd [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+pidfile="/var/run/$SVCNAME.pid"
+
+depend()
+{
```
Comment (jirutka) [inline testing/shairport-sync/shairport-sync.initd]: Please move move `{` up, i.e. `depend() {`.

File: testing/shairport-sync/shairport-sync.initd (hunk 3 above)
Comment (henriknil) [inline testing/shairport-sync/shairport-sync.initd]: shairport-sync crashes if I try to make openrc daemonize it.

**Pull request closed without merging**